package.json

```diff
@@ -10,18 +10,18 @@
     "access": "public"
   },
   "devDependencies": {
-    "@types/express": "4.17.11",
-    "@types/jest": "26.0.23",
-    "@types/lodash": "4.14.168",
     "@types/redis": "2.8.28",
```

Comment:
Can updated to v3.x (at less)

Reply:
There is no version 3.X.X. That is the issue. Also, `@types/redis` is deprecated for the new redis versions